src/DurableTask.SqlServer/DurableTask.SqlServer.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.4.*" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.5.1" />
```

Comment:
Version **2.5.1** is not yet on nuget.org. We're currently pulling it from our myget feed.